app/furniture/marketplace/vendor_stripe_accounts_controller.rb

```diff
@@ -1,5 +1,5 @@
 class Marketplace
-  class StripeAccountsController < FurnitureController
+  class VendorStripeAccountsController < FurnitureController
```

Comment:
I'm not sold on this rename, since I feel like `StripeAccount` is the actual domain model here; and the `vendor_stripe_account` is an indicator of who the `StripeAccount` in the `Marketplace` is connected to. We could wind up with `distributor_stripe_account` for use cases where Arizmendi works with Piikup and the local bicycle courier coop.